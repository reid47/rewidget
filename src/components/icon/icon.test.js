import React from 'react';
import { render } from 'enzyme';
import Icon from './icon';

describe('Icon', () => {
  describe('name = check', () => {
    test('default size', () => {
      expect(render(<Icon name="check" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="check" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="check" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon
            name="check"
            title="Test icon title"
            desc="Test icon description"
          />
        )
      ).toMatchSnapshot();
    });
  });

  describe('name = chevron_down', () => {
    test('default size', () => {
      expect(render(<Icon name="chevron_down" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="chevron_down" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="chevron_down" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon
            name="chevron_down"
            title="Test icon title"
            desc="Test icon description"
          />
        )
      ).toMatchSnapshot();
    });
  });

  describe('name = chevron_left', () => {
    test('default size', () => {
      expect(render(<Icon name="chevron_left" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="chevron_left" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="chevron_left" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon
            name="chevron_left"
            title="Test icon title"
            desc="Test icon description"
          />
        )
      ).toMatchSnapshot();
    });
  });

  describe('name = chevron_right', () => {
    test('default size', () => {
      expect(render(<Icon name="chevron_right" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="chevron_right" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="chevron_right" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon
            name="chevron_right"
            title="Test icon title"
            desc="Test icon description"
          />
        )
      ).toMatchSnapshot();
    });
  });

  describe('name = chevron_up', () => {
    test('default size', () => {
      expect(render(<Icon name="chevron_up" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="chevron_up" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="chevron_up" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon
            name="chevron_up"
            title="Test icon title"
            desc="Test icon description"
          />
        )
      ).toMatchSnapshot();
    });
  });

  describe('name = chevron_up_down', () => {
    test('default size', () => {
      expect(render(<Icon name="chevron_up_down" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="chevron_up_down" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="chevron_up_down" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon
            name="chevron_up_down"
            title="Test icon title"
            desc="Test icon description"
          />
        )
      ).toMatchSnapshot();
    });
  });

  describe('name = menu', () => {
    test('default size', () => {
      expect(render(<Icon name="menu" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="menu" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="menu" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon
            name="menu"
            title="Test icon title"
            desc="Test icon description"
          />
        )
      ).toMatchSnapshot();
    });
  });

  describe('name = search', () => {
    test('default size', () => {
      expect(render(<Icon name="search" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="search" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="search" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon
            name="search"
            title="Test icon title"
            desc="Test icon description"
          />
        )
      ).toMatchSnapshot();
    });
  });

  describe('name = to_end', () => {
    test('default size', () => {
      expect(render(<Icon name="to_end" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="to_end" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="to_end" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon
            name="to_end"
            title="Test icon title"
            desc="Test icon description"
          />
        )
      ).toMatchSnapshot();
    });
  });

  describe('name = to_start', () => {
    test('default size', () => {
      expect(render(<Icon name="to_start" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="to_start" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="to_start" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon
            name="to_start"
            title="Test icon title"
            desc="Test icon description"
          />
        )
      ).toMatchSnapshot();
    });
  });

  describe('name = x', () => {
    test('default size', () => {
      expect(render(<Icon name="x" />)).toMatchSnapshot();
    });

    test('custom size', () => {
      expect(
        render(<Icon name="x" width="36" height="36" />)
      ).toMatchSnapshot();
    });

    test('custom className', () => {
      expect(
        render(<Icon name="x" className="test-class" />)
      ).toMatchSnapshot();
    });

    test('custom title and desc', () => {
      expect(
        render(
          <Icon name="x" title="Test icon title" desc="Test icon description" />
        )
      ).toMatchSnapshot();
    });
  });
});
