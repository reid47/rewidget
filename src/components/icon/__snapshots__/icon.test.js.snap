// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon name = check custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="20 6 9 17 4 12"
  />
</svg>
`;

exports[`Icon name = check custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="20 6 9 17 4 12"
  />
</svg>
`;

exports[`Icon name = check custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="20 6 9 17 4 12"
  />
</svg>
`;

exports[`Icon name = check default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="20 6 9 17 4 12"
  />
</svg>
`;

exports[`Icon name = chevron_down custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="4 9 12 17 20 9"
  />
</svg>
`;

exports[`Icon name = chevron_down custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="4 9 12 17 20 9"
  />
</svg>
`;

exports[`Icon name = chevron_down custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="4 9 12 17 20 9"
  />
</svg>
`;

exports[`Icon name = chevron_down default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="4 9 12 17 20 9"
  />
</svg>
`;

exports[`Icon name = chevron_left custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="15 4 7 12 15 20"
  />
</svg>
`;

exports[`Icon name = chevron_left custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="15 4 7 12 15 20"
  />
</svg>
`;

exports[`Icon name = chevron_left custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="15 4 7 12 15 20"
  />
</svg>
`;

exports[`Icon name = chevron_left default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="15 4 7 12 15 20"
  />
</svg>
`;

exports[`Icon name = chevron_right custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="9 4 17 12 9 20"
  />
</svg>
`;

exports[`Icon name = chevron_right custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="9 4 17 12 9 20"
  />
</svg>
`;

exports[`Icon name = chevron_right custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="9 4 17 12 9 20"
  />
</svg>
`;

exports[`Icon name = chevron_right default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="9 4 17 12 9 20"
  />
</svg>
`;

exports[`Icon name = chevron_up custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="4 15 12 7 20 15"
  />
</svg>
`;

exports[`Icon name = chevron_up custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="4 15 12 7 20 15"
  />
</svg>
`;

exports[`Icon name = chevron_up custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="4 15 12 7 20 15"
  />
</svg>
`;

exports[`Icon name = chevron_up default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="4 15 12 7 20 15"
  />
</svg>
`;

exports[`Icon name = chevron_up_down custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="6 9 12 3 18 9"
  />
  <polyline
    points="6 15 12 21 18 15"
  />
</svg>
`;

exports[`Icon name = chevron_up_down custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="6 9 12 3 18 9"
  />
  <polyline
    points="6 15 12 21 18 15"
  />
</svg>
`;

exports[`Icon name = chevron_up_down custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="6 9 12 3 18 9"
  />
  <polyline
    points="6 15 12 21 18 15"
  />
</svg>
`;

exports[`Icon name = chevron_up_down default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="6 9 12 3 18 9"
  />
  <polyline
    points="6 15 12 21 18 15"
  />
</svg>
`;

exports[`Icon name = menu custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="2 6 22 6"
  />
  <polyline
    points="2 12 22 12"
  />
  <polyline
    points="2 18 22 18"
  />
</svg>
`;

exports[`Icon name = menu custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="2 6 22 6"
  />
  <polyline
    points="2 12 22 12"
  />
  <polyline
    points="2 18 22 18"
  />
</svg>
`;

exports[`Icon name = menu custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="2 6 22 6"
  />
  <polyline
    points="2 12 22 12"
  />
  <polyline
    points="2 18 22 18"
  />
</svg>
`;

exports[`Icon name = menu default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="2 6 22 6"
  />
  <polyline
    points="2 12 22 12"
  />
  <polyline
    points="2 18 22 18"
  />
</svg>
`;

exports[`Icon name = search custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="11"
    cy="11"
    r="9"
  />
  <line
    x1="22"
    x2="17.65"
    y1="22"
    y2="17.65"
  />
</svg>
`;

exports[`Icon name = search custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="11"
    cy="11"
    r="9"
  />
  <line
    x1="22"
    x2="17.65"
    y1="22"
    y2="17.65"
  />
</svg>
`;

exports[`Icon name = search custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="11"
    cy="11"
    r="9"
  />
  <line
    x1="22"
    x2="17.65"
    y1="22"
    y2="17.65"
  />
</svg>
`;

exports[`Icon name = search default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="11"
    cy="11"
    r="9"
  />
  <line
    x1="22"
    x2="17.65"
    y1="22"
    y2="17.65"
  />
</svg>
`;

exports[`Icon name = to_end custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="7 4 15 12 7 20"
  />
  <polyline
    points="18 4 18 20"
  />
</svg>
`;

exports[`Icon name = to_end custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="7 4 15 12 7 20"
  />
  <polyline
    points="18 4 18 20"
  />
</svg>
`;

exports[`Icon name = to_end custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="7 4 15 12 7 20"
  />
  <polyline
    points="18 4 18 20"
  />
</svg>
`;

exports[`Icon name = to_end default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="7 4 15 12 7 20"
  />
  <polyline
    points="18 4 18 20"
  />
</svg>
`;

exports[`Icon name = to_start custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="17 4 9 12 17 20"
  />
  <polyline
    points="6 4 6 20"
  />
</svg>
`;

exports[`Icon name = to_start custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="17 4 9 12 17 20"
  />
  <polyline
    points="6 4 6 20"
  />
</svg>
`;

exports[`Icon name = to_start custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="17 4 9 12 17 20"
  />
  <polyline
    points="6 4 6 20"
  />
</svg>
`;

exports[`Icon name = to_start default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <polyline
    points="17 4 9 12 17 20"
  />
  <polyline
    points="6 4 6 20"
  />
</svg>
`;

exports[`Icon name = x custom className 1`] = `
<svg
  class="test-class"
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <line
    x1="18"
    x2="6"
    y1="6"
    y2="18"
  />
  <line
    x1="6"
    x2="18"
    y1="6"
    y2="18"
  />
</svg>
`;

exports[`Icon name = x custom size 1`] = `
<svg
  fill="none"
  height="36"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="36"
  xmlns="http://www.w3.org/2000/svg"
>
  <line
    x1="18"
    x2="6"
    y1="6"
    y2="18"
  />
  <line
    x1="6"
    x2="18"
    y1="6"
    y2="18"
  />
</svg>
`;

exports[`Icon name = x custom title and desc 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <line
    x1="18"
    x2="6"
    y1="6"
    y2="18"
  />
  <line
    x1="6"
    x2="18"
    y1="6"
    y2="18"
  />
</svg>
`;

exports[`Icon name = x default size 1`] = `
<svg
  fill="none"
  height="24"
  role="img"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <line
    x1="18"
    x2="6"
    y1="6"
    y2="18"
  />
  <line
    x1="6"
    x2="18"
    y1="6"
    y2="18"
  />
</svg>
`;
